cypress/integration/providers_spec.js

```diff
@@ -110,6 +110,7 @@ describe('Dashboard Providers Page', () => {
     });
 
     it('should edit a provider', () => {
+      // TODO [MHS, 2020-03-24]  UH. what's going on there. with submit then success loop.
```

Comment:
I would like to fix this, but not in this PR.  Unless someone demands it.

Reply:
Not sure what that's referring to, but please create a new issue, if it makes sense to do so.